### PR TITLE
Add index, AUR package, rearrange subheaders

### PR DESCRIPTION
**Pull request by 0xf0xx0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +28,15 @@ TODO:
 * Compiler
 * Webasm target
 
-## Code example:
+# Installation
+`UwUpp` can be installed on Arch-based systems with `yay uwupp-git`. For all other systems:
```
Comment (Deltaphish) [inline README.md]: Clarify that the package is on AUR, in case someone uses some other client instead of `yay`, or wants to install it manualy.

**Review by Deltaphish — CHANGES_REQUESTED** [review]
Thanks for fixing a AUR package! 99% of UwU++ users probably run arch so this should save allot of headache.
Apart from one nitpick everything looks good, will merge as soon as I have double checked that everything is working on my machine. I think my arch install is broken however, so it might take a day or two.

File: README.md (hunk 1 above)
Comment (0xf0xx0) [reply README.md]: thats a good point, lemmie edit that

**Review by Deltaphish — APPROVED** [review]
Works on my arch laptop

**Pull request merged**